src/display/display_utils.js

```diff
@@ -67,14 +67,15 @@ class FilterFactory {
 
   get #defs() {
     if (!this.#_defs) {
+      const div = this.#document.createElement("div");
+      div.setAttribute("id", "svgDefinitions");
```

Comment:
The `FilterFactory` functionality is exposed in the *general* API, hence a solution that requires the default viewer CSS-file in order to work unfortunately doesn't really work.
While it's not as nice, I do believe that you'll need to set the CSS rules inline here.